exonum/src/blockchain/service.rs

```diff
@@ -518,6 +535,25 @@ impl SharedNodeState {
             .reconnects_timeout
             .remove(addr)
     }
+
+    /// Sets an address of WebSocket server.
+    pub(crate) fn set_server_address(&self, address: Addr<Syn, WsServer>) {
```

Comment:
`SharedStateApi` is a broker object, this why looking on method signature is not clear what `server_address` is setting there.